augur/routes/contributor_reports.py

```diff
@@ -356,11 +356,14 @@ def get_repo_id_start_date_and_end_date():
 
         now = datetime.datetime.now()
 
-        repo_id = int(request.args.get('repo_id'))
+        repo_id = request.args.get('repo_id')
         start_date = str(request.args.get('start_date', "{}-01-01".format(now.year - 1)))
         end_date = str(request.args.get('end_date', "{}-{}-{}".format(now.year, now.month, now.day)))
 
-        return repo_id, start_date, end_date
+        if repo_id:
+            return int(repo_id), start_date, end_date
+
```

Comment:
is the none, none, none API return set what generates the "Hey, you need a repo ID there pardner?" message.

Reply:
Yes, basically if the repo_id is not in the query params, then it returns None for all return variables. I could have just returned None, but I decided to still return None for the start_date and end_date too.